### PR TITLE
Fix vaultpw command

### PR DESCRIPTION
**Pull request by JonathanRenon-EDB**
##### SUMMARY
Update vaultpw to match the new expected parser declaration.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
vaultpw

##### ADDITIONAL INFORMATION
ansible-vaultpw command would fail with error:

```
ERROR! Unexpected Exception, this is probably a bug: cannot import name 'optparse_helpers' from 'ansible.cli.arguments' (/home/ams/work/2ndq/ansible/lib/ansible/cli/arguments/__init__.py)
the full traceback was:

Traceback (most recent call last):
  File "/home/ams/work/2ndq/ansible/bin/ansible-vaultpw", line 92, in <module>
    mycli = getattr(__import__("ansible.cli.%s" % sub, fromlist=[myclass]), myclass)
  File "/home/ams/work/2ndq/ansible/lib/ansible/cli/vaultpw.py", line 13, in <module>
    from ansible.cli.arguments import optparse_helpers as opt_help
ImportError: cannot import name 'optparse_helpers' from 'ansible.cli.arguments' (/home/ams/work/2ndq/ansible/lib/ansible/cli/arguments/__init__.py)
```

### CODE REVIEW

**Comment by matthbakeredb** [comment]
Are the changes in this PR authored by you @JonathanRenon-EDB or are they a merge from upstream?

**Comment by JonathanRenon-EDB** [comment]
those changes are authored by me since it's on vaultpw.py which isn't part of ansible upstream.

File: lib/ansible/cli/vaultpw.py [hunk 1]
```diff
@@ -19,113 +19,138 @@
...
 
-        if len(context.CLIARGS['args']) != 1:
+        if len(context.CLIARGS["args"]) != 1:
             raise AnsibleOptionsError("Exactly one inventory file must be specified")
 
-        self.file = os.path.expanduser(context.CLIARGS['args'][0])
+        self.file = os.path.expanduser(context.CLIARGS["args"][0])
 
-        old_umask = os.umask(0o077)
+        context.CLIARGS["func"]()
```
Comment (matthbakeredb) [inline lib/ansible/cli/vaultpw.py]: Move this line down
```suggestion
```

File: lib/ansible/cli/vaultpw.py [hunk 2]
```diff
@@ -19,113 +19,138 @@
...
 
-        self.execute()
+        old_umask = os.umask(0o077)
```
Comment (matthbakeredb) [inline lib/ansible/cli/vaultpw.py:129]:
```suggestion
        old_umask = os.umask(0o077)
        context.CLIARGS["func"]()
```

**Pull request closed without merging**